src/main/java/seedu/ui/UI.java

```diff
@@ -118,28 +118,19 @@ public void deleteEventMessage(String eventType, String eventName) {
                 + "your Event list.\n", eventType, eventName);
     }
 
-    public void printInvalidDateTimeFormat() {
```

Comment:
why are you deleting this? Just need to verify here

Reply:
this is from merge conflict.